orca-igor/src/main/java/com/netflix/spinnaker/orca/igor/tasks/GetBuildPropertiesTask.java

```diff
@@ -62,6 +62,9 @@
             stageDefinition.getMaster(),
             stageDefinition.getJob());
     if (properties.size() == 0) {
```

Comment:
while you're in here, do you mind converting that to this?

```suggestion     
    if (properties.isEmpty()) {
```